### PR TITLE
Upgrade Releases tab rewith the update center info then augment with release notes

### PR DESCRIPTION
**Pull request by halkeye**
Adding a archives tab which lists past releases.

Its a fairly early pass and UI is not my specialty

### CODE REVIEW

**Comment by halkeye** [comment]
~Demo: https://jenkins-plugins.g4v.dev~

**Comment by halkeye** [comment]
Demo: https://plugin-site-regud.ondigitalocean.app/

File: src/components/PluginReleases.jsx [hunk 1]
```diff
@@ -2,52 +2,65 @@ import React, {useState, useEffect} from 'react';
...
+                                    <TimeAgo date={new Date(version.buildDate)} formatter={formatter}/>
                                 </div>
                             </h5>
                         </div>
                         <div className="card-body">
-                            <p
+                            <div>
+                                <InstallViaCLI pluginId={pluginId} version={version.version} />
+                            </div>
+                            {release.bodyHTML && (<blockquote
```
Comment (timja) [inline src/components/PluginReleases.jsx:56]: I'm not a huge fan of this block quote bit, I think we should remove it,

especially I don't like the italics
https://plugin-site-regud.ondigitalocean.app/slack/#releases

image

Comment (halkeye) [reply src/components/PluginReleases.jsx:56]: > I'm not a huge fan of this block quote bit, I think we should remove it,

and replace it with? just the \<p\> directly doesn't really work, it blends everything together, though open to ideas with spacing and stuff

Comment (timja) [reply src/components/PluginReleases.jsx:56]: what if we just move the install via cli to after the notes and before the checksums?

**Comment by halkeye** [comment]
Merging, and we'll revisit the design in a future PR

File: src/templates/plugin.jsx [hunk 2]
```diff
@@ -73,14 +73,13 @@ function PluginPage({data: {jenkinsPlugin: plugin, reverseDependencies: reverseD
...
                             {plugin.wiki.content && <div className="content" dangerouslySetInnerHTML={{__html: plugin.wiki.content}} />}
                         </>)}
-                        {state.selectedTab === 'releases' && <PluginReleases pluginId={plugin.id} />}
+                        {state.selectedTab === 'releases' && <PluginReleases pluginId={plugin.id} versions={versions.edges.map(edge => edge.node)} />}
                         {state.selectedTab === 'issues' && <PluginIssues pluginId={plugin.id} />}
                         {state.selectedTab === 'dependencies' && <PluginDependencies dependencies={plugin.dependencies} reverseDependencies={reverseDependencies.edges}/>}
                     </div>
                 </div>
                 <div className="col-md-3 gutter">
-                    <a href={`https://updates.jenkins.io/download/plugins/${plugin.name}`}
```
Comment (daniel-beck) [inline src/templates/plugin.jsx:82]: A second redundant link to the same tab is redundant.

Finding this out during advisory publication is 👎

**Pull request merged**